.eslintignore

```diff
@@ -0,0 +1 @@
+flow-typed/
```

Comment:
This was painful: they're not proper JS files so:
* 1. Eslint was outputting errors at commit time (like "needs the comment @flow if you use Flow types")
* 2. prettier kept rewriting them even when I wanted to make only a small change